Packs/CommonScripts/Scripts/IsNotInCidrRangeCopy/IsNotInCidrRangeCopy.py

```diff
@@ -0,0 +1,58 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+import ipaddress
+
+
+def validate_cidr(cidr: str):
+    """
+    Validates CIDR format.
+    """
+    try:
+        ipaddress.ip_network(cidr)
+
+    except ValueError as e:
+        demisto.debug(f'Skipping "{cidr}": {e}')
+        return False
+
+    return True
+
+
+def main():
+    """ Check if given IP (or IPs) address is part of a given CIDR (or a list of CIDRs).
+
+    Args:
+        ip_addresses (str): A list of IPs/IPv6s
+        cidr_range_list (str): A list of CIDRs to be checked against.
+
+    Returns:
+        bool: True if given IP is part of given CIDR range.
+    """
+
+    ip_addresses = argToList(demisto.args()['left'])
+    cidr_range_list = argToList(demisto.args()['right'])
+
+    try:
+
+        not_in_range = True
+
+        for ip in ip_addresses:
+
+            try:
+                ip = ipaddress.ip_address(ip)
+
+            except ValueError as e:
+                demisto.debug(f'Skipping "{ip}": {e}')
+                continue
+
+            not_in_range = all(ip not in ipaddress.ip_network(cidr) for cidr in cidr_range_list if validate_cidr(cidr))
```

Comment:
The `not_in_range` variable is overwritten in each iteration of the loop over `ip_addresses`, which means that the function will only return the result for the last IP address in the list, not for all of them.

You could create a list to store the results for each IP address, and then return whether all IP addresses are not in the CIDR range:
```python
not_in_range_list = []
for ip in ip_addresses:
    ...
    not_in_range_list.append(all(ip not in ipaddress.ip_network(cidr) for cidr in cidr_range_list if validate_cidr(cidr)))